src/app/gateways/routes.ts

```diff
@@ -35,6 +35,11 @@ export const routes = () => {
             name: 'gateway-clusters-view',
             component: () => import('@/app/data-planes/views/DataPlaneClustersView.vue'),
           },
+          {
+            path: 'config',
+            name: 'gateway-config-view',
+            component: () => import('@/app/gateways/views/GatewayConfigView.vue'),
+          },
```

Comment:
Is this missing currently? Ideally this would be a separate PR if so.

Reply:
Yes. See https://github.com/kumahq/kuma-gui/pull/1701.